### ISSUE TITLE
CLI --react results in 400 error

### ISSUE THREAD

**Issue by simasch** [vaadin/hilla#692]
**This is a blocker!**

```
npx @hilla/cli init --react --auth react-with-auth but get Unable to create project: 400
```

Trying to create a React-based Hilla app results in a 400 error.

**Comment by platosha** [comment]
Hi, thanks for reporting. Sadly the `--auth` option in combination with React is not supported yet. Does it work without `--auth` for you?

**Comment by simasch** [comment]
@platosha Yes it works without --auth. A better error message would be great!
